src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="page__calendar calendar calendar--start-day--sun calendar--month-length-29">
+      <div class="calendar__cell calendar__cell--1"></div>
+      <div class="calendar__cell calendar__cell--2"></div>
+      <div class="calendar__cell calendar__cell--3"></div>
+      <div class="calendar__cell calendar__cell--4"></div>
+      <div class="calendar__cell calendar__cell--5"></div>
+      <div class="calendar__cell calendar__cell--6"></div>
```

Comment:
Do not hard code day number for cell. use [:nth-child ](https://css-tricks.com/how-nth-child-works/)
`page__calendar` - is redudant. Use it styles in `calendar`
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length-29">
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
```